app/helpers/dateFormat.js

```diff
@@ -1,9 +1,45 @@
 import { format } from "util";
+import { getGlobalCfg } from "config";
```

Comment:
I'd try to avoid using `getGlobalCfg` here.

The helpers module is intended to be as self contained as possible, so introducing the getGlobalCfg here is not great.

One way to remove this dependency would be the following: 

- Introduce the `dateToLocal`, `dateToUTC` as you did
- Create a `tsToDate` function in the `selectors.js` file. That would get from the currently saved config state and return the correct function (either dateToLocal or dateToUTC)
- Refactor the uses of `tsToDate` (they're currently either in components or on actions) to get from the selector.

@alexlyp thoughts on this? The above will be slighly more refactoring work but would drop the dependency to config here. Do you think it's worth doing?